Files/ViewModels/ItemViewModel.cs

```diff
@@ -1671,14 +1671,14 @@ private async void ProcessOperationQueue(CancellationToken cancellationToken)
             const uint FILE_ACTION_RENAMED_OLD_NAME = 0x00000004;
             const uint FILE_ACTION_RENAMED_NEW_NAME = 0x00000005;
 
-            var sampler = new IntervalSampler(500);
+            var sampler = new IntervalSampler(200);
```

Comment:
Did you test what kind of performance impact this change has?

Reply:
I've tried pasting and deleting a lot of files from a folder and it seemed alright.